src/machine/board_teensy40.go

```diff
@@ -0,0 +1,177 @@
+// +build teensy40
+
+package machine
+
+const (
+	NUM_DIGITAL_IO_PINS = 40
+	NUM_ANALOG_IO_PINS  = 14
+)
```

Comment:
Are these used anywhere? If not, I would suggest removing them (I suspect they are necessary in the Arduino environment).

Reply:
Nope, not used, will remove